IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcAnnotation/DocEntity.xml

```diff
@@ -6,12 +6,12 @@
 		<DocLocalization Locale="fr" Name="Annotation" />
 	</Localization>
 	<Attributes>
-		<DocAttribute Name="ContainedInStructure" UniqueId="ee617cc7-7598-48b6-bbbf-fc39ddd793b1" DefinedType="IfcRelContainedInSpatialStructure" AggregationType="3" AggregationLower="0" AggregationUpper="1" Inverse="RelatedElements">
-			<Documentation>Relationship to a spatial structure element, to which the associate is primarily associated.</Documentation>
-		</DocAttribute>
 		<DocAttribute Name="PredefinedType" UniqueId="f0de9d01-18e5-4fdd-9a5c-c0a195481766" DefinedType="IfcAnnotationTypeEnum" AttributeFlags="1">
 			<Documentation></Documentation>
 		</DocAttribute>
+		<DocAttribute Name="ContainedInStructure" UniqueId="ee617cc7-7598-48b6-bbbf-fc39ddd793b1" DefinedType="IfcRelContainedInSpatialStructure" AggregationType="3" AggregationLower="0" AggregationUpper="1" Inverse="RelatedElements">
+			<Documentation>Relationship to a spatial structure element, to which the associate is primarily associated.</Documentation>
+		</DocAttribute>
```

Comment:
I am not sure, what this change influences. Just the order of some sort?

Reply:
As the commit [d721044](https://github.com/bSI-InfraRoom/IFC-Specification/pull/349/commits/d721044c410720062f8b0831551bfc530346c360) description says.